### DISCUSSION TITLE
Optimization Error [main.12345...js] Expected identifier but found "="

### DISCUSSION THREAD

**Discussion by papagei-ma** [serverless/serverless#10485] [Q&A]
### Are you certain it's a bug?

- [X] Yes, it looks like a bug

### Is the issue caused by a plugin?

- [X] It is not a plugin issue

### Are you using the latest version?

- [X] Yes, I'm using the latest version

### Is there an existing issue for this?

- [X] I have searched existing issues, it hasn't been reported yet

### Issue description

When trying to upgrade to 2.71.0, the issue is there.
I have an ionic on angular project, and use serverless to manage (AWS (appsync, graphql ..)  related services. And when I tried to use the current version from (serverless package) I started getting this error. 
Error: Optimization error [main.16fb586baab212b0.js]: Error: Transform failed with 1 error:
main.16fb586baab212b0.js:13285:5: ERROR: Expected identifier but found "="
    at failureErrorWithLog (/project/node_modules/esbuild/lib/main.js:1493:15)
......
...... 

### Service configuration (serverless.yml) content

```yaml
I dont think its related to the configuration, because the bug is at build time.
```

### Command name and used flags

ionic build ( which will run ng build)

### Command output

```shell
Error: Optimization error [main.16fb586baab212b0.js]: Error: Transform failed with 1 error:
main.16fb586baab212b0.js:13285:5: ERROR: Expected identifier but found "="
```

### Environment information

```shell
Serverless: Running "serverless" installed locally (in service node_modules)
Framework Core: 2.71.0 (local)
Plugin: 5.5.3
SDK: 4.3.0
Components: 3.18.1
```

**Comment by medikoo** [comment]
@papagei-ma `esbuild`, in stack trace suggests you're relying on `serverless-esbuild` plugin`. If you feel there's an issue with it, please report it at its repository https://github.com/floydspace/serverless-esbuild

I'm moving this to discussions

**Comment by zakhenry** [comment]
@papagei-ma did you end up working out what the issue was here?
